bucket/sqlcl.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "22.2.0.172.1718",
+    "description": "Oracle SQL Developer Command Line (SQLcl)",
+    "homepage": "https://www.oracle.com/database/sqldeveloper/technologies/sqlcl/",
+    "license": "Oracle Free Use License",
```

Comment:
```suggestion
    "license": {
        "identifier": "Oracle Free Use License",
        "url": "https://www.oracle.com/downloads/licenses/oracle-free-license.html"
    },
```

Reply:
i will test, there is issue with checkver

Reply:
Checkver seems fine, but PowerShell seems to be unable to load the URL. Any ideas @issaclin32?

Reply:
Yeah. `wget` could not load the page due to cookie storage permissions.
It is totally fine for users who open the page with a browser, but there could be potential problems in automated workflows.

How about using the snapshot from **archive.org**?
https://web.archive.org/web/20220615160155/https://www.oracle.com/downloads/licenses/oracle-free-license.html

Reply:
> How about using the snapshot from archive.org?

For checkver? The archive.org pages will not get updated, right?

Reply:
Oh. I thought you were talking about the license page.